Server/mods/deathmatch/logic/lua/CLuaManager.cpp

```diff
@@ -233,6 +233,7 @@ void CLuaManager::LoadCFunctions()
         {"getServerPassword", CLuaFunctionDefs::GetServerPassword},
         {"setServerPassword", CLuaFunctionDefs::SetServerPassword},
         {"getServerConfigSetting", CLuaFunctionDefs::GetServerConfigSetting},
+        {"getServerEnvironmentValue", CLuaFunctionDefs::GetServerEnvironmentValue},
```

Comment:
```cpp
 {"getServerEnvironmentValue", ArgumentParser<GetServerEnvironmentValue>}
```

Also, for this to work, you need to move it to `CLuaUtilDefs`.
The new parser doesn't work in `CLuaManager`.